Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -31,8 +31,12 @@ public class BTShopperInsightsClient {
     /// - Returns: A `BTShopperInsightsResult` instance
     /// - Note: This feature is in beta. It's public API may change or be removed in future releases.
     public func getRecommendedPaymentMethods(request: BTShopperInsightsRequest) async throws -> BTShopperInsightsResult {
+        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.recommendedPaymentsStarted)
```

Comment:
yeah that's a great callout - im fine with either but feel more inclined to leave as is since it seems the other clients (i.e. `BTCardClient`) define analytic helper methods for only success, failure and cancel and for start events, they fire it like above.

Reply:
We only call started 1 time per flow so I think we can continue to leave it without extracting it into its own method as @agedd stated.